### ISSUE TITLE
Empty line after version not ignored

### ISSUE THREAD

**Issue by smilingthax** [tapmodo/node-ldif#1]
This generates a parse error with node-ldif, while being accepted by other ldif software:
```
console.log(require('ldif').parse(
  'version: 1\n' +
  '\n' +    // <-- commenting out this line makes it work again
  'dn: dc=test\n' +
  'dc: test\n'
));
```

**Comment by tapmodo** [comment]
Thank you! I think you are right, that this is valid. I'll try to verify that with the RFC but I think you are right, this is acceptable by other parsers. I will add it to the tests, create and issue a fix, and update this ticket.

**Comment by tapmodo** [comment]
This should be working now, thanks for submitting this issue!  
Published an updated version 0.5.0 to npm.
